src/test/java/io/thinkit/edc/client/connector/services/PresentationsTest.java

```diff
@@ -0,0 +1,64 @@
+package io.thinkit.edc.client.connector.services;
+
+import io.thinkit.edc.client.connector.model.*;
+
+/*class PresentationsTest extends PresentationApiTestBase {
```

Comment:
If we aren't able to run tests for documentation issue I'd suggest to disable them through the `@Disabled` annotation instead, so they will be reported in the test execution